benchalerts/tests/unit_tests/expected_md/summary_noruns.md

```diff
@@ -0,0 +1,3 @@
+Conbench analyzed the 0 benchmark runs that triggered this notification.
+
+None of the specified runs were found on the Conbench server. See the [build logs](https://austin.something) for more information.
```

Comment:
❤️ for linking out to where one might find a way to figure out what's up. That's fantastic.